homeassistant/components/snips.py

```diff
@@ -81,6 +107,13 @@ def message_received(topic, payload, qos):
             _LOGGER.error('Received invalid JSON: %s', payload)
             return
 
+        if (request['intent']['probability']
+                < config[DOMAIN].get(CONF_PROBABILITY)):
```

Comment:
default for probability is 0. So if none is configured, nothing will ever pass? Better to not have a probability default and just check if probability is None or intentProb < probability

Reply:
Default is checked for greater than, so 0 matches all. It's a lower limit not upper.

Reply:
The config option defaults to 0.
Snips probability is between 0 and 1.

If not configured, probability defaults to 0.

So you're checking here things like `if 0.8 < 0`. Which will always be false?

Also, if you know a key exists, prefer to use square brackets over `get`

Reply:
0.8 > 0

Reply:
Ah, I see it now. My bad.